cmd/hooks/prestop.go

```diff
@@ -28,33 +27,30 @@ No unnecessary delay is added to the termination workflow, as the PreStop hook l
 If the PreStop hook hangs during its execution, the driver node pod will be forcefully terminated after 32 seconds (30s terminationGracePeriodSeconds + 2 second grace period extension from Kubelet).
```

Comment:
I don't think this comment will always be accurate post PR, because terminationGracePeriodSeconds might not always be 30s? Maybe change to "terminationGracePeriodSeconds in the pod specification define the upper bound for timeout".

Reply:
+1, consider adding the words something like "by default" at the end

Reply:
Good call, updated comment